### PR TITLE
This PR adds an example to test the method wait_for_simulation_step_to_end()

### PR DESCRIPTION
**Pull request by juanjqo**
-passing-passing)
Static Badge
@dqrobotics/developers 

Hi @mmmarinho, 

This PR adds a new minimal example of the synchronous mode and test the method `wait_for_simulation_step_to_end()` added in [PR_51](https://github.com/dqrobotics/python/pull/51).

Best regards, 

Juancho

### CODE REVIEW

File: vrep_interface_tests/synchronous_test/synch_test.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+"""
+
+from dqrobotics.interfaces.vrep import DQ_VrepInterface
+import time
+
+
+def main() -> None:
+    vi = DQ_VrepInterface()
+    try:
+        vi.connect("127.0.0.1", 19997, 100, 10)
```
Comment (mmmarinho) [inline vrep_interface_tests/synchronous_test/synch_test.py]: @juanjqo Please test for a valid connection and raise an Exception if it does not connect.

File: vrep_interface_tests/synchronous_test/synch_test.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            y_sim = vi.get_object_pose("/Sphere").translation().vec3()[2]
+            y_est = y_0 - 0.5 * 9.81 * pow(t, 2)
+            vi.trigger_next_simulation_step()
+            vi.wait_for_simulation_step_to_end()
+        print("Elapsed time: ", t)
+        print("Estimated height: ", y_est, "Measured height: ", y_sim)
+        vi.stop_simulation()
+        vi.disconnect()
+
+    except KeyboardInterrupt:
```
Comment (mmmarinho) [inline vrep_interface_tests/synchronous_test/synch_test.py]: @juanjqo 

For this try--catch I'd do something similar to what I wrote below. The reason is that you'll need to disconnect even after a `KeyboardInterrupt` and for now you have it repeated in both the regular flow and the `exception Exception as e` clause. That seems like a textbook case for using `finally`.

```python
try: 
    # Your code up till line 51, without the stop_simulation etc
except (Exception, KeyboardInterrupt):
    pass
finally:
    vi.stop_simulation()
    vi.disconnect()
```

**Review by mmmarinho — CHANGES_REQUESTED** [review]

File: vrep_interface_tests/synchronous_test/synch_test.py (hunk 2 above)
Comment (juanjqo) [reply vrep_interface_tests/synchronous_test/synch_test.py]: @mmmarinho I updated the example as you suggested (assuming I'm not wrong). However, the script now does not show the error to the user. For instance, in a scene without the object `/Sphere` the output is:
```shell
Process finished with exit code 0
```

File: vrep_interface_tests/synchronous_test/synch_test.py (hunk 1 above)
Comment (juanjqo) [reply vrep_interface_tests/synchronous_test/synch_test.py]: @mmmarinho I added the test, as suggested.

File: vrep_interface_tests/synchronous_test/synch_test.py (hunk 2 above)
Comment (mmmarinho) [reply vrep_interface_tests/synchronous_test/synch_test.py]: @juanjqo Thanks, it was an example... so if you'd like to print the Exception you can use

```python
except (Exception, KeyboardInterrupt) as e:
    print(e)
```

or with a formatted error output with more information.

Comment (juanjqo) [reply vrep_interface_tests/synchronous_test/synch_test.py]: @mmmarinho thanks for the examples. I updated the script.

**Review by mmmarinho — APPROVED** [review]

**Pull request merged**